ports/stm32/pyb_can.c

```diff
@@ -376,8 +387,35 @@ STATIC MP_DEFINE_CONST_FUN_OBJ_1(pyb_can_state_obj, pyb_can_state);
 // Get info about error states and TX/RX buffers
 STATIC mp_obj_t pyb_can_info(size_t n_args, const mp_obj_t *args) {
     #if MICROPY_HW_ENABLE_FDCAN
-    // TODO implement for FDCAN
-    return mp_const_none;
+    // Done: check implementation for FDCAN
```

Comment:
I guess this comment can be removed now, because it works?

Reply:
Indeed, it can. I will put it on my todo list. Next to the G4 PR...